examples/mujoco/plotter.py

```diff
@@ -3,14 +3,20 @@
 import argparse
 import os
 import re
+from typing import Any
 
 import matplotlib.pyplot as plt
 import matplotlib.ticker as mticker
 import numpy as np
 from tools import csv2numpy, find_all_files, group_files
 
 
-def smooth(y, radius, mode="two_sided", valid_only=False):
+def smooth(
+    y: np.ndarray,
+    radius: int,
+    mode: str = "two_sided",
```

Comment:
pls use Literal as type, then you can remove the assert

Reply:
Yes. it's cleaner.